core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -173,6 +197,54 @@ public static int numCommonBlockingTaskThreads() {
     }
 
     /**
+     * Returns the default maximum length of a request. Note that this value has effect only if a user
+     * did not specify it.
```

Comment:
Could you explicitly state which side these new flags affect? e.g. Returns the server-side default maximum length of a request.